pkgs/servers/foundationdb/cmake.nix

```diff
@@ -58,18 +64,22 @@ let
             # Same with LLD when Clang is available.
             (lib.optionalString useClang    "-DUSE_LD=LLD")
             (lib.optionalString (!useClang) "-DUSE_LD=GOLD")
+          ] ++ lib.optionals (lib.versionOlder version "7.0.0")
+          [ # FIXME: why can't libressl be found automatically?
+            "-DLIBRESSL_USE_STATIC_LIBS=FALSE"
+            "-DLIBRESSL_INCLUDE_DIR=${ssl.dev}"
+            "-DLIBRESSL_CRYPTO_LIBRARY=${ssl.out}/lib/"
+            "-DLIBRESSL_SSL_LIBRARY=${ssl.out}/lib/"
+            "-DLIBRESSL_TLS_LIBRARY=${ssl.out}/lib/"
+          ] ++ lib.optionals (lib.versionAtLeast version "7.1.0" && lib.versionOlder version "7.2.0")
+          [ # FIXME: why can't libressl be found automatically?
+            "-DOPENSSL_USE_STATIC_LIBS=FALSE"
+            "-DOPENSSL_CRYPTO_LIBRARY=${ssl.out}/lib/"
+            "-DOPENSSL_SSL_LIBRARY=${ssl.out}/lib/"
           ];
 
         inherit patches;
 
-        # fix up the use of the very weird and custom 'fdb_install' command by just
-        # replacing it with cmake's ordinary version.
-        postPatch = ''
-          for x in bindings/c/CMakeLists.txt fdbserver/CMakeLists.txt fdbmonitor/CMakeLists.txt fdbbackup/CMakeLists.txt fdbcli/CMakeLists.txt; do
```

Comment:
There are more files using `fdb_install` now, but instead of updating this snippet it seems that we don't need to patch this at all?